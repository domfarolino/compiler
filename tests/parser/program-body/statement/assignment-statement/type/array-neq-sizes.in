program dominic is

  string target[0: 12];
  string expression[24: 31];

begin

  target := expression;

end program.
