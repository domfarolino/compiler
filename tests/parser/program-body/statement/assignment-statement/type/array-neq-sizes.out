[1;31mLine 8: Assignment statement target and expression arrays must be of the same length
Line 8: Error parsing program body
[0m